bgcval2/analysis_timeseries.py

```diff
@@ -1669,26 +1662,26 @@ def convertmeqm3TOumolkg(nc, keys):
         av[name]['datadetails'] = {'name': '', 'units': ''}
         av[name]['dataFile'] = ''
         av[name]['datasource'] = ''
-        if analysisSuite.lower() == 'fast':
-            av[name]['Dimensions'] = 2
-            av[name]['modeldetails'] = {
-                'name': name,
-                'vars': [
-                    'OCN_PH',
-                ],
-                'convert': ukp.NoChange,
-                'units': 'pH',
-            }
-        else:
-            av[name]['Dimensions'] = 3
-            av[name]['modeldetails'] = {
-                'name': name,
-                'vars': [
-                    'PH3',
-                ],
-                'convert': ukp.NoChange,
-                'units': 'pH',
-            }
+#        if analysisSuite.lower() == 'fast':
+#            av[name]['Dimensions'] = 2
+#            av[name]['modeldetails'] = {
+#                'name': name,
+#                'vars': [
+#                    'OCN_PH',
+#                ],
+#                'convert': ukp.NoChange,
+#                'units': 'pH',
+#            }
+#        else:
```

Comment:
pls remove commented out code

Reply:
This whole section is in the 4000 lines that are scheduled to be deleted over the next few PRs. I'll remove it then!

Reply:
the more you remove (obv if needs to be removed) the less spaghetti the code becomes so your next PR will be cleaner to solve :)